br/pkg/storage/local.go

```diff
@@ -95,6 +96,11 @@ func (l *LocalStorage) WriteFile(_ context.Context, name string, data []byte) er
 	if err := os.Rename(tmpPath, filepath.Join(l.base, name)); err != nil {
 		return errors.Trace(err)
```

Comment:
comment for `WriteFile`: maybe check the `name` does not contain path separator. Because it will cause the file to be created outside of the folder, and sync the folder inode is not enough

Reply:
Perhaps we can sync the dirpath. 🤔